arviz/plots/backends/bokeh/forestplot.py

```diff
@@ -191,8 +202,10 @@ def plot_forest(
     ] = -all_plotters[  # pylint: disable=protected-access
         0
     ].group_offset
-    axes[0, 0].y_range._property_values["end"] = y_max  # pylint: disable=protected-access
+    axes[0, 0].y_range._property_values["end"] = y_max  # pylint: disable=protected-
```

Comment:
undo this change to have pyling keep ignoring this warning

Reply:
Did this, still checks are failing:-C